index.bs

```diff
@@ -5171,6 +5175,7 @@ type.
         "DOMException" Null
         BufferRelatedType Null
         "FrozenArray" "&lt;" Type "&gt;" Null
+        "dictionary" "&lt;" Type "," Type "&gt;" Null
```

Comment:
You probably want to give it its own non-terminal to account for the first Type being optional:

```html
<pre class="grammar" id="prod-OpenDictionaryType">
    OpenDictionaryType :
        "dictionary" "&lt;" Type "&gt;"
        "dictionary" "&lt;" Type, Type "&gt;"
</pre>
```